### PR TITLE
chore(BackgroundImage): update tests

### LINKED ISSUE #9525: Background image

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #9525

### CODE REVIEW

File: packages/react-core/src/components/BackgroundImage/__tests__/BackgroundImage.test.tsx [hunk 1]
```diff
@@ -2,19 +2,27 @@ import React from 'react';
...
+  render(<BackgroundImage src="/image/url.png" data-testid="test-id" />);
+  expect(screen.getByTestId('test-id')).toHaveClass('pf-v5-c-background-image', { exact: true });
+});
+
+test('has src URL applied to style', () => {
   render(<BackgroundImage src="/image/url.png" data-testid="test-id" />);
 
-  const backgroundImage = screen.getByTestId('test-id');
-  const backgroundImageStyle = backgroundImage.getAttribute('style');
+  const backgroundImageStyle = screen.getByTestId('test-id').getAttribute('style');
```
Comment (thatblindgeye) [inline packages/react-core/src/components/BackgroundImage/__tests__/BackgroundImage.test.tsx:13]: Nit: we could remove this variable altogether and just have the expectation be `expect(screen.getByTestId('test-id')).toHaveAttribute('style', '--pf-v5-c-background-image--BackgroundImage:/image/url.png')` (give or take a space between the CSS var and value)

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
One nit below. We should also use the `styles` object for class names, and add a test for spreading additional props

**Comment by kmcfaul** [comment]
> We should also use the `styles` object for class names

What do you mean here? 

I've updated with your other feedback.

**Comment by kmcfaul** [comment]
Updated to use the styles object.

**Review by thatblindgeye — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

**Pull request merged**